Makefile

```diff
@@ -16,134 +17,93 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #*******************************************************************************
-
 ifeq ($(BOLOS_SDK),)
-$(error BOLOS_SDK is not set)
+$(error Environment variable BOLOS_SDK is not set)
 endif
 
-dummy_submodules := $(shell git submodule update --init --recursive)
+include $(BOLOS_SDK)/Makefile.defines
 
-ifeq ($(TARGET_NAME),TARGET_NANOS)
-APP_STACK_SIZE:=880
-endif
+########################################
+#        Mandatory configuration       #
+########################################
 
-include $(BOLOS_SDK)/Makefile.defines
+# Enabling DEBUG flag will enable PRINTF and disable optimizations
+#DEBUG = 1
 
-# Main app configuration
-APPNAME="Binance Chain"
-APPVERSION_M=1
-APPVERSION_N=1
-APPVERSION_P=7
+# Application name
+APPNAME = "Binance Chain"
 
-APP_LOAD_PARAMS = --appFlags 0x200 --delete $(COMMON_LOAD_PARAMS) --path "44'/714'"
+# Application version
+APPVERSION_M= 1
+APPVERSION_N= 1
+APPVERSION_P= 7
```

Comment:
Maybe a new version should be set, at least because you have changed the `zxlib`, and adapt the different part of the app?

Reply:
Done.